.github/workflows/swan_pr_workflow.yml

```diff
@@ -0,0 +1,146 @@
+# GitHub action file for SwanHub SDK
+name: SwanHub SDK Application Workflow
+
+# Triggers the workflow on push or pull request
+on:
+  push: # Triggered on a push to any branch ( do not change )
+  schedule:
+    # Run workflow at 4 AM EST (9 AM UTC)
+    - cron: '0 9 * * *'
+    # Run workflow at 7 PM EST (12 AM UTC next day)
+    - cron: '0 0 * * *'
+  release:
+    types: [ created ] # Triggers the workflow when a release is created.
+
+# Defines the jobs that the workflow will execute
+jobs:
+  Lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install black
+        run: |
+          pip install black
+      - name: Lint with Black
+        run: |
+          black --check ./**/*.py
+  Test:
+    needs: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install pytest
+        run: |
+          pip install -r requirements.txt
```

Comment:
not too sure if these have been tested but this may have to be pip3

Reply:
most developers use `alias pip='pip3'` in their zsh/bash file for ease of use

Reply:
but it make sense